### ISSUE TITLE
Add real time altitude when driving and etc

### ISSUE THREAD

**Issue by utnet9684** [organicmaps/organicmaps#9159]
I like organic maps a lot, but a feature I miss a lot especially when driving/biking around mountains is real time altitude. It would be amazing to have this feature.

**Comment by biodranik** [comment]
Tap your current position on the map (blue arrow) to see the altitude and speed.

**Comment by utnet9684** [comment]
> Tap your current position on the map (blue arrow) to see the altitude and speed.

Yeah, but not very safe to be tapping every now and then when on the road, especially on mountains small roads...

**Comment by RedAuburn** [comment]
what's your use case?

**Comment by utnet9684** [comment]
> what's your use case?

checking the altitude while driving in the alps and mountains in general

**Comment by muralito** [comment]
> > what's your use case?
> 
> checking the altitude while driving in the alps and mountains in general

Sure. It's wonderful to see the number grows when you are driving and climbing a mountain.

For a trip it's also useful to show the time and distance already traveled, and the average speed of this travel.

**Comment by utnet9684** [comment]
> > > what's your use case?
> > 
> > 
> > checking the altitude while driving in the alps and mountains in general
> 
> Sure. It's wonderful to see the number grows when you are driving and climbing a mountain.
> 
> For a trip it's also useful to show the time and distance already traveled, and the average speed of this travel.

that would be great also, like osmand+ that already has those features. But organic has a much cleaner view and consumes much less battery. Altitude is also helpfull when walking in the mountains.

**Comment by muralito** [comment]
My old nuvi has a customization feature they call "Dashboards". 
You can choose from a dashboard with simple and prominent display of current speed and arrival time, to others that show driving direction (not degrees, just 2 letters with the cardinal point) , distance to destination, elevation. 
All using the bottom of the screen. Some are opaque, others are semitransparent.
All dashboards always shows the speed, and from 1 to 3 other values. 
Remember that if is device with low res screen. 
When you are in driving mode if you tap the "speed" field, it takes you to a screen like a travel computer, with trips A and B, it shows a big speedometer and driving direction, and other travel data, like max speed, moving avg speed, overall speed, stopped time, moving time, and total time.
If you tap other fields of the dashboard it offers to select between which data you want to display in that box.

Other unrelated features, are a "mileage report" (i don't remember if the device does it detecting when the car is providing 12V or changes in position/speed), "ecoChallenge" which is a gamified feature to helps you drive saving fuel, it gives you an score according to speed, braking and acceleration, and you can see the score live in the dashboard with a green leaf which turns yellow or red according to the score, and a time series view of score the last 20 hours of driving.

**Comment by flomnes** [comment]
Maybe organicmaps could have a "cyclist/athletic" mode ? Where indicators such as speed, altitude, etc. are displayed on screen and updated in real time.

**Comment by brdbkr** [comment]
I would like this feature in my car and on the main app, not just in bicycle mode.

Ideally I'd like to see an optional status bar with speed, compass orientation, altitude, and state/province/county/etc.

34 mph - NNW - 1180 ft - Fulton Co, GA 30008

53 kph - 165° - 312m - Landes France

**Comment by utnet9684** [comment]
> Maybe organicmaps could have a "cyclist/athletic" mode ? Where indicators such as speed, altitude, etc. are displayed on screen and updated in real time.
> 
> Of course, this could be turned on/off using a global navigation setting. Or enabled only if the uses chooses the "bicycle" icon when planning a route.
> 
> @utnet9684 @muralito would that work out for you ?

that would be cool, but also displayed when driving an automobile.

**Comment by Rob-from-VI** [comment]
Totally agree that altitude display ( on phone screen and in android auto screen) while navigating in car, bicicling, or hiking, would be a wonderful addition.

**Comment by brdbkr** [comment]
Tapping the arrow does nothing in CarPlay.

**Comment by soc7** [comment]
that's a feature I would appreciate too

**Comment by Rob-from-VI** [comment]
As mentioned above Android Auto, similar to carplay, is also lacking this functionality so far for me.

It would be very nice to have.
